cluster-up/cluster/kind-1.22-sriov/sriov-components/manifests/multus/kustomization.yaml

```diff
@@ -5,3 +5,10 @@ resources:
 images:
 - name: nfvpe/multus
   newName: quay.io/kubevirtci/multus
+patchesJson6902:
+- path: patch-args.yaml
```

Comment:
lets call the file with a more generic name maybe?
like `patches.yaml` in case we need more fields, no need per field file (i.e args here)

Reply:
I followed similar example (the once that patch sriov-cni and sriov-dp manifests) pattern where there is patch file
for each domain (e.g: args, node selector, etc..) with the name `patch-<the domain of the change>.yaml`.
Initially I called it `patch-logs-args.yaml` then I renamed to `patch-args.yaml` to be a little bit more generic.